### PR TITLE
Add description for StylePropertyMapReadOnly::getAll

### PR DESCRIPTION
**Pull request by nainar**
@shans PTAL? Thanks!

### CODE REVIEW

File: css-typed-om/Overview.bs [hunk 1]
```diff
@@ -280,6 +280,29 @@ The <dfn method for=StylePropertyMap>append(DOMString <var>property</var>,
...
+        
+        1. If |property| is a [=list-valued property=],
+        return the list of |values| assigned to that property.
+        
+        2. If |property| is not a [=list-valued property=],
+        let |values| initially be the empty [=list=] and append 
+        the |value| assigned to the |property| to this list 
+        and return |values|.
+    
+    4. Else, return `null`.
```
Comment (shans) [inline css-typed-om/Overview.bs]: Not sure if this should be null or []. @tabatkins ? At any rate, good to merge.

Comment (tabatkins) [inline css-typed-om/Overview.bs]: Checking with @annevk and @domenic (who have also worked with developing the MultiMap pattern) to see if `getAll()` should return `undefined` or `[]` for a non-existent key.  (It definitely shouldn't be `null`, tho.)

Comment (annevk) [inline css-typed-om/Overview.bs]: Answer is [].

**Comment by nainar** [comment]
Changed `null` to empty list. Merging change in now.

**Pull request merged**